### PR TITLE
Monday

### PR DESCRIPTION
**Pull request by Dijana181**

### CODE REVIEW

File: src/main/scala/Recursion1.scala [hunk 1]
```diff
@@ -0,0 +1,10 @@
+object Recursion1 extends App {
+    var count = 0
+
+    def recur(word:String,times:Int){
```
Comment (tvaidotas) [inline src/main/scala/Recursion1.scala:4]: var count = 0 shouldn't be a variable and instead be in the method like this

def recur(word:String,times:Int, count: Int){

and then as you'll be calling it just increment it by one

File: src/main/scala/Recursion2.scala [hunk 2]
```diff
@@ -0,0 +1,16 @@
+object Recursion2 extends App {
+
+  var count = 0
+
+  def recur(word:String,times:Int){
```
Comment (tvaidotas) [inline src/main/scala/Recursion2.scala:5]: same as in the previous comment

File: src/main/scala/Recursion3.scala [hunk 3]
```diff
@@ -0,0 +1,25 @@
+object Recursion3 extends App {
+  var count = 1
+
+  def recur(three:String,five:String,times:Int){
```
Comment (tvaidotas) [inline src/main/scala/Recursion3.scala:4]: same

File: src/main/scala/TooHot.scala [hunk 4]
```diff
@@ -0,0 +1,9 @@
+object TooHot extends App{
+
+  def climate (temperature:Int, isSummer:Boolean):Unit = {
+    if (isSummer==true && temperature > 60 && temperature<=100) print("true")
```
Comment (tvaidotas) [inline src/main/scala/TooHot.scala:4]: isSummer is a boolean by itself so you don't need to do == true against it, you could do 
if (isSummer && temperature > 60 && temperature<=100)

File: src/main/scala/TooHot.scala [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+    else if (isSummer==false && temperature >60 && temperature<=90 ) print("true")
```
Comment (tvaidotas) [inline src/main/scala/TooHot.scala:5]: !isSummer could replace isSummer==false

Comment (Dijana181) [reply src/main/scala/TooHot.scala:5]: Ok, thanks. I didn't know that you could use this syntax

**Pull request closed without merging**